src/renderer/src/routes/index.test.tsx

```diff
@@ -0,0 +1,75 @@
+import type { ReactNode } from 'react'
+import { getDeviceInfoQueryKey } from '@comapeo/core-react'
+import { QueryClient } from '@tanstack/react-query'
+import { RouterProvider } from '@tanstack/react-router'
+import { render, waitFor } from '@testing-library/react'
+import { describe, expect, test } from 'vitest'
+
+import { router } from '../App'
```

Comment:
same comment about creating these router instances within the tests instead of using the singleton instance